easybuild/easyconfigs/k/king/king-2.2.4.eb

```diff
@@ -0,0 +1,29 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'BinariesTarball'
+
+name = 'king'
+version = '2.2.4'
+local_version_shorthand = '224'
```

Comment:
I would remove the dots from `version` on the fly instead of using an additional variable for it (see below)
```suggestion
```

Reply:
As above, the line was removed